Libplanet.Net.Tests/TestUtils.cs

```diff
@@ -85,5 +95,84 @@ public static PrivateKey GeneratePrivateKeyOfBucketIndex(Address tableAddress, i
 
             return privateKey;
         }
+
+        public static BlockChain<DumbAction> CreateDummyBlockChain(MemoryStoreFixture fx)
+        {
+            var store = new MemoryStore();
+            var blockChain = new BlockChain<DumbAction>(
+                TestUtils.Policy,
+                new VolatileStagePolicy<DumbAction>(),
+                store,
+                new TrieStateStore(new MemoryKeyValueStore()),
+                fx.GenesisBlock);
+
+            return blockChain;
+        }
+
+        public static (ITransport, ConsensusContext<DumbAction>, PrivateKey)
```

Comment:
`PrivateKey` is argument of the function and is returned. And also for the transport, how about pass it to the argument and make creator method for `ITransport` in `TestUtils.cs`?